hamster_lib/helpers/time.py

```diff
@@ -142,11 +149,11 @@ def date_time_from_groupdict(groupdict):
 
     # Individual patterns for time/date  substrings.
     relative_pattern = '(?P<relative>-\d+)'
-    time_pattern = '(?P<time>\d{2}:\d{2})'
+    time_pattern = '(?P<time>\d{2}:\d{2}(?<seconds>:\d{2})?)'
     date_pattern = '(?P<date>\d{4}-\d{2}-\d{2})'
-    datetime_pattern = '(?P<datetime>\d{4}-\d{2}-\d{2} \d{2}:\d{2})'
+    datetime_pattern = ('(?P<datetime>\d{4}-\d{2}-\d{2} \d{2}:\d{2}(:\d{2})?)')
```

Comment:
Yes, the parentheses were used for string concatenation. Since there is only one string now, they are not needed.